### ISSUE TITLE
Panes are not resizable for some layouts

### ISSUE THREAD

**Issue by Z35hLy48k** [zellij-org/zellij#2829]
## Problem

When executing zellij with a custom layout shown below, if the pane size is specified with numbers(size=20), the panes cannot be resized.
But if the pane size are specified with percentages (size="20%"), they can be resized.

**Expected behavior:**
Be able to resize panes even after specifying pane sizes with layout 

## Other information
$ zellij -l test.kdl 

test.kdl
```
layout {
    pane
    pane size=20
}
```

```
$ alacritty --version
alacritty 0.12.0

$ stty size
53 95

$ uname -av
Linux machine 6.5.5 #1-NixOS SMP PREEMPT_DYNAMIC Sat Sep 23 09:14:39 UTC 2023 x86_64 GNU/Linux
```
## Logs

[zellij.log](https://github.com/zellij-org/zellij/files/12778193/zellij.log)

**Comment by imsnif** [comment]
Yeah, these are fixed sized panes. This is the intended behavior...
